### ISSUE TITLE
Question about the zero-shot settings

### ISSUE THREAD

**Issue by ODRLver** [chinnarouge/CXR_CML#1]
Extremely impressive work! After carefully reading the content of the paper, I mainly have one question. Since I am not very sure about the complete definition of zero-shot, but I noticed that you used the labels and data of CXR-LT for training in the training code, I would like to understand how zero-shot is reflected in your article? Looking forward to your reply!

Oh! I guess the zero-shot here is that we freeze the CLIP text encoder and visual encoder to train only the classifier?
